src/main/java/me/botsko/prism/Prism.java

```diff
@@ -344,9 +344,11 @@ public void onEnable() {
         }
         if (getConfig().getBoolean("prism.paste.enable")) {
             pasteKey = Prism.config.getString("prism.paste.api-key", "API KEY");
-            if (pasteKey != null && pasteKey.startsWith("API key")) {
+            if (pasteKey != null && pasteKey.startsWith("API key") || pasteKey.length() < 6) {
```

Comment:
Should this be `&& !pasteKey.startsWith("API key")` ?

Reply:
nope thats the default...which we want to make null

Reply:
basically that line filters out bad api keys sort of